src/runtime_src/core/tools/common/ReportMechanical.cpp

```diff
@@ -18,35 +18,31 @@
 // Local - Include Files
 #include "ReportMechanical.h"
 
+#include "core/common/sensor.h"
+

```

Comment:
I don't think we need json_parser.hpp anymore

Reply:
I removed it.  Thanks!